__tests__/createBrowserLikeFetch.spec.js

```diff
@@ -294,8 +340,9 @@ describe('createCookiePassingFetch', () => {
     const hostname = 'hello.example.com';
 
     const enhancedFetch = createBrowserLikeFetch({ hostname, setCookie })(mockFetch);
-    await enhancedFetch('https://example.org/api/some-resource', { credentials: 'include' });
+    await enhancedFetch('https://example.com/api/some-resource', { credentials: 'include' });
```

Comment:
appeared to be a mis-match in the domain, I think this change is correct but deserves some scrutiny nonetheless